plugin.video.arloview/resources/lib/arlo_stream.py

```diff
@@ -4,6 +4,7 @@
 import urllib
 import urlparse
```

Comment:
This module doesn't exist in Python 3 with this name anymore.
In future updates your add-on has to be fully Python 3 compatible.